src/platform/code-editor/code-editor.component.ts

```diff
@@ -616,5 +668,7 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
     window.addEventListener('resize', () => {
         this._editor.layout();
     });
+    this.addFullScreenModeCommand();
   }
+
```

Comment:
please also add a test for this feature here:
https://github.com/Teradata/covalent-code-editor/blob/develop/src/platform/code-editor/code-editor.component.spec.ts

Reply:
Let's not have these be private methods.  The F11 keybinding is good but shouldn't be the only way to go full screen.  And then lets add the unit test and the API documentation.  👍

Reply:
I will change it to public methods.
F11 is an extra keybinding for fullscreen. On Actual user can go to fullscreen mode by context menu.
Please find attached screen shot.
I will be adding unit tests for public methods.
Regarding API documentation, there is no extra propert/event added in editor. So API documentation change wont be required.

![image](https://user-images.githubusercontent.com/4894148/49440530-f1256300-f7e9-11e8-89da-8f0558cadc81.png)